### ISSUE TITLE
There is a stylistic alternative für a, but none for ä

### ISSUE THREAD

**Issue by do18** [bBoxType/FiraGO#17]
Using InDesign CS2 though, so maybe it's not FiraGO's fault at all.

**Comment by carrois** [comment]
You are right, we quickly implemented the alternative /a on request because it was there anyway (uni0251). But we didn’t go for all the /a diacritics at that time.
